src/control/cam_control_mod.meta

```diff
@@ -0,0 +1,32 @@
+[ccpp-table-properties]
+  name = cam_control_mod
+  type = module
+
+[ccpp-arg-table]
+  name  = cam_control_mod
+  type  = module
+[ aqua_planet ]
+  standard_name = flag_for_aqua_planet_mode
```

Comment:
A conversation starter for @nusbaume - There is one `flag_for` standard name and a bunch of `do` in the standard names repo.  An alternate name could be `do_aqua_planet`

Reply:
ooh i like `do_aqua_planet` better. I've changed it to that, but am happy to hear Jesse's feedback when he returns!